.github/workflows/main.yml

```diff
@@ -1,15 +1,18 @@
 name: CI
 
-on: [pull_request]
+on: [push, pull_request]
```

Comment:
I would rather stay with only pull as then the tests are trigered twice for usual pr

Reply:
There would be also additional trigger after the merge, which IMO is good, as between the creation of pull request and the final merge some time usually pass, during which some dependencies can have new releases, etc.; in perfect world the tests would be retriggered just before the merge, but that's often not the case, so I see this actually as an added value (catch potential issues earlier).

But it's up to you – please feel free to change it directly here if you prefer :wink: